### PR TITLE
Feature/completeUsernameCopy

### PR DESCRIPTION
**Pull request by NassimBounouas**
This pull concerns #1843 and add a button on account page to copy a complete username.

### CODE REVIEW

File: client/src/app/+accounts/accounts.component.ts [hunk 1]
```diff
@@ -70,4 +70,18 @@ export class AccountsComponent implements OnInit, OnDestroy {
           )
     }
   }
+
+  copyText(val: string){
```
Comment (NassimBounouas) [inline client/src/app/+accounts/accounts.component.ts]: Declaration of public instance method not allowed after declaration of private instance method.

Comment (NassimBounouas) [reply client/src/app/+accounts/accounts.component.ts]: Fixed

**Comment by Chocobozzz** [comment]
Thanks for the PR @NassimBounouas 

Could you use [ngx-clipboard](https://github.com/maxisam/ngx-clipboard)? Like we do in the [share modal](https://github.com/Chocobozzz/PeerTube/blob/develop/client/src/app/videos/%2Bvideo-watch/modal/video-share.component.html#L17)

File: client/src/assets/images/global/copy.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (Chocobozzz) [inline client/src/assets/images/global/copy.html]: This is a [softies](https://robbiepearce.com/softies/) icon?

Comment (NassimBounouas) [reply client/src/assets/images/global/copy.html]: No it's not, I will change it !

**Comment by NassimBounouas** [comment]
As you suggested @Chocobozzz, I replaced the icon by the `glyphicon-copy` and the copy function has been replaced by ngx-clipboard component. If you have any other suggestion, I'll do it with pleasure.

**Comment by Chocobozzz** [comment]
Thanks @NassimBounouas!

**Pull request merged**